Makefile

```diff
@@ -30,80 +30,80 @@ html-with-api: Makefile
 	@echo Step 1: Creating html files
 	$(SPHINXBUILD) $(SPHINXOPTS) $(SOURCEDIR) $(BUILDDIR)/html
 	@echo Step 2: Building API
-	./make_help_scripts/create_api
+	./make_help_scripts/create_api.py
 
 html-all-subrepos: Makefile
 	@echo Single html file without API
 	@echo Step 1: Cloning all subrepositories
-	./make_help_scripts/add_sub_repos
+	./make_help_scripts/add_sub_repos.py
 	@echo Step 2: Building documentation
 	$(SPHINXBUILD) $(SPHINXOPTS) $(SOURCEDIR) $(BUILDDIR)/html
 	@echo Step 3: Deleting subrepositories in doc/ folder
-	./make_help_scripts/delete_sub_repos
+	./make_help_scripts/delete_sub_repos.py
 
 html-all-subrepos-with-errors: Makefile
 	@echo Single html file without API
 	@echo Step 1: Cloning all subrepositories
-	./make_help_scripts/add_sub_repos
+	./make_help_scripts/add_sub_repos.py
 	@echo Step 2: Building documentation
 	$(SPHINXBUILD) $(SPHINXOPTS) -W --keep-going $(SOURCEDIR) $(BUILDDIR)/html
 	@echo Step 3: Deleting subrepositories in doc/ folder
-	./make_help_scripts/delete_sub_repos
+	./make_help_scripts/delete_sub_repos.py
 
 html-all-subrepos-with-api: Makefile
 	@echo Single html file with API
 	@echo Step 1: Cloning all subrepositories
-	./make_help_scripts/add_sub_repos
+	./make_help_scripts/add_sub_repos.py
 	@echo Step 2: Building documentation
 	$(SPHINXBUILD) $(SPHINXOPTS) $(SOURCEDIR) $(BUILDDIR)/html
-	@echo Step 3: Deleting subrepositories in doc/ folder
-	./make_help_scripts/delete_sub_repos
-	@echo Step 4: Building API
-	./make_help_scripts/create_api
+	@echo Step 3: Building API
+	./make_help_scripts/create_api.py
+	@echo Step 4: Deleting subrepositories in doc/ folder
+	./make_help_scripts/delete_sub_repos.py
 
 linkcheck-all-subrepos-with-api: Makefile
 	@echo Single version html file with API and linkcheck
 	@echo Step 1: Cloning all subrepositories
-	./make_help_scripts/add_sub_repos
+	./make_help_scripts/add_sub_repos.py
 	@echo Step 2: Building API
-	./make_help_scripts/create_api
-	@echo Step 3: Cloning all subrepositories again
-	./make_help_scripts/add_sub_repos
```

Comment:
Don't we need this cloning subrepositories again script?

Reply:
No, because I removed the cleanup from the `create_api` script

Reply:
Awesome!